platform_channels/lib/src/accelerometer_event_channel.dart

```diff
@@ -15,23 +15,23 @@ class Accelerometer {
   static Stream<AccelerometerReadings> get readings {
     return _eventChannel.receiveBroadcastStream().map(
           (dynamic event) => AccelerometerReadings(
-            event[0] as double,
-            event[1] as double,
-            event[2] as double,
+            event[0] as double?,
+            event[1] as double?,
+            event[2] as double?,
```

Comment:
Can we implicitly downcast it to be double? This way we can avoid using bang operator event_channel_demo.dart, and properties in AccelerometerReadings can be NNBD.

Reply:
> Can we implicitly downcast it to be `double`? This way we can avoid using bang operator in event_channel_demo.dart, and properties in AccelerometerReadings can be NNBD.

I am not sure @AyushBherwani1998  how implicitly downcast will work here.

Reply:
It was a mistake, sorry. I mean explicitly downcasting it to be a `double` instead of `double?`

Reply:
> It was a mistake, sorry. I mean explicitly downcasting it to be a `double` instead of `double?`

Okay, @AyushBherwani1998  Should  I go for tweaking the changes and update the PR accordingly?

Reply:
I'll suggest to wait for some one from the team to review and give their thoughts to avoid rework.